ci-operator/step-registry/cucushift/upgrade/eus/unpause-worker-mcp/cucushift-upgrade-eus-unpause-worker-mcp-commands.sh

```diff
@@ -19,32 +19,75 @@ function debug() {
     fi
 }
 
+# After upgrading, the paused mcp should not be upgraded, so its version is not same with normal mcp
+function pre_check_mcp(){
+    echo "Pre check, after upgrading, the paused mcp should not be upgraded..."
+    local paused_worker_version running_worker_version
+    paused_worker_version=$(oc get -l 'node-role.kubernetes.io/'${PAUSED_MCP_NAME}'=' nodes -ojson | jq '.items[0].status.nodeInfo' | jq -r '.containerRuntimeVersion,.kernelVersion,.osImage')
```

Comment:
if there are more that one node in the paused pool, we should check all the nodes's version, right?

Reply:
I checked step `cucushift-upgrade-toimage`, it only checked `oc adm upgrade` and history. let me execute `check_mcp ${PAUSED_MCP_NAME}` before `pre_check_mcp` to make sure mcp already updated.

Reply:
check_mcp only guarantee the mcp status, right? per your [comment](https://github.com/openshift/release/pull/55328/files#r1722784213), we still need check node's version, so here the concern is that `items[0]` could not represent all nodes in the pool, we should loop all the nodes for the check.

Reply:
@rioliu-rh could you help confirm that if all nodes in a mcp has same version when mcp is ready.